### ISSUE TITLE
Any way to automatically camelCase all keys?

### ISSUE THREAD

**Issue by Fire-Dragon-DoL** [amatsuda/jb#12]
Hello! In ruby we use snake_case everywhere and I suffer when I have to camelCase in any place.
Is there any option to camelKeys automatically all keys right before spitting out the JSON?

I would use it so that my JSON api stays consistent with javascript notation

**Comment by vassilevsky** [comment]
We are solving this problem in our projects with https://github.com/RSCSCybersecurity/sparrow and it works pretty well.

**Comment by frederikspang** [comment]
Or in case you wanna avoid using an extra middleware (Which isn't a bad solution, but some might prefer not to), you could in your builder-files use `.camelize(:lower)` for the keys, if you're settings the attributes dynamically.

**Comment by ksouthworth** [comment]
I agree with @Fire-Dragon-DoL it would be great to have support for automatically camelCasing keys for output.

Something like the `olive_branch` gem https://github.com/vigetlabs/olive_branch

**Comment by andrewclink** [comment]
I was using a MultiJson.use(MyCustomKeyTransformer) to do this, and it worked pretty well. Unfortunately a JB commit in October seems to have broken multi_json support. Multi_json does provide a nice two-way API, though, if the situation is solvable.
